goSrc/src/vmCaller/iroha/commands.go

```diff
@@ -640,7 +641,8 @@ func handleErrors(result *C.Iroha_CommandError, err error, commandName string) (
 		if error_extra_ptr != nil {
 			error_extra = ": " + *error_extra_ptr
 		}
-		return fmt.Errorf("Error executing %s command: %s", commandName, error_extra)
+		IrohaErrorDetails = fmt.Sprintf("%s %s error_code %d ",commandName, error_extra, result.error_code)
```

Comment:
```suggestion
		IrohaErrorDetails = fmt.Sprintf("%s %s error_code %d", commandName, error_extra, result.error_code)
```

Reply:
I don't know golang, but isn't `IrohaErrorDetails` global variable? If it is - is it sure that multiple smart contracts would save the value in the same time?

Reply:
It seems it is global. But all the fields are reinitialized in VmCall for each ```c++ -> go``` call, before the other calls made from ```go``` part.